### PR TITLE
Express ranges in mathematical notation

### LINKED ISSUE #412: @armfazh: draft 11: In prose, write `[start, stop)` instead of `range(start, stop)`

### PR DESCRIPTION
**Pull request by cjpatton**
Closes #412.

* Use `[0, end)` instead of `range(end)`
* Use `[start, end)` instead of `range(start,end)`
* Use `[start, end]` instead of `range(start,end+1)`
* Use `^` to denote exponentiation rather than `**`

### CODE REVIEW

**Comment by cjpatton** [comment]
Rebased.

File: draft-irtf-cfrg-vdaf.md [hunk 1]
```diff
@@ -1019,7 +1019,7 @@ accomplished using the following algorithm:
 
   Pre-conditions:
 
-    * `agg_id` MUST be in `range(SHARES)` and match the index of
+    * `agg_id` MUST be in `[0,SHARES)` and match the index of
```
Comment (divergentdave) [inline draft-irtf-cfrg-vdaf.md:1022]: nit: consistent spacing
(this recurs in several places throughout the diff)
```suggestion
    * `agg_id` MUST be in `[0, SHARES)` and match the index of
```

**Review by divergentdave — COMMENTED** [review]
FWIW there are some more `**`s in the document still, in more complicated expressions:

* Section 2, under `to_be_bytes()` and `to_le_bytes()`
* Section 7.4.2, definition of `offset` (though this and the preceding line look like one-line Python snippets to me)
* Two more instances in the following paragraph
* Section 7.4.5, definition of `offset` (again, these look like one-line Python snippets, so maybe we want to leave this as-is)
* Section 9.6, in the artwork

I'd also suggest using "... in the range `[0, x)`" over "... in range `[0, x)`" for stylistic reasons.

File: draft-irtf-cfrg-vdaf.md [hunk 2]
```diff
@@ -2584,7 +2584,7 @@ of proofs.
 This section specifies `Prio3`, an implementation of the `Vdaf` interface
 ({{vdaf}}). It has three generic parameters: an `NttField ({{ntt-field}}), an
```
[the remark this answers is not in the document]
Comment (cjpatton) [reply draft-irtf-cfrg-vdaf.md:2585]: Nice catch

**Comment by cjpatton** [comment]
> FWIW there are some more `**`s in the document still, in more complicated expressions:
> 
>     * Section 2, under `to_be_bytes()` and `to_le_bytes()`
> 
>     * Section 7.4.2, definition of `offset` (though this and the preceding line look like one-line Python snippets to me)
> 
>     * Two more instances in the following paragraph
> 
>     * Section 7.4.5, definition of `offset` (again, these look like one-line Python snippets, so maybe we want to leave this as-is)
> 
>     * Section 9.6, in the artwork
> 
> 
> I'd also suggest using "... in the range `[0, x)`" over "... in range `[0, x)`" for stylistic reasons.

Whoops, I missed this before I re-requested review. I'll address this now.

**Review by divergentdave — APPROVED** [review]

**Comment by cjpatton** [comment]
Rebased and squashed.

**Pull request merged**